### ISSUE TITLE
Broken `@oasisprotocol/sapphire-hardhat` installation

### ISSUE THREAD

**Issue by matevz** [oasisprotocol/sapphire-paratime#264]
##### SUMMARY

Following https://docs.oasis.io/dapp/sapphire/quickstart#init-a-new-hardhat-project fails when invoking

```shell
pnpm add -D @oasisprotocol/sapphire-hardhat
```

```
 ERR_PNPM_WORKSPACE_PKG_NOT_FOUND  In : "@oasisprotocol/sapphire-paratime@workspace:^" is in the dependencies but no package named "@oasisprotocol/sapphire-paratime" is present in the workspace

This error happened while installing the dependencies of @oasisprotocol/sapphire-hardhat@2.17.3

Packages found in the workspace: 
Progress: resolved 3, reused 1, downloaded 1, added 0
```

Similarly, `yarn add -D @oasisprotocol/sapphire-hardhat` requires from you to select a version, which seems awkward:

```
[1/4] Resolving packages...
Couldn't find any versions for "@oasisprotocol/sapphire-paratime" that matches "workspace:^"
? Please choose a version of "@oasisprotocol/sapphire-paratime" from this list: (Use arrow keys)
❯ 1.2.2 
  1.1.4 
  1.1.3 
  1.1.1 
  1.1.0 
  1.0.15 
  1.0.14 
  1.0.13 
  1.0.11 
  1.0.10 
  1.0.9 
  1.0.7 
  1.0.6 
  1.0.5 
  1.0.4 
  1.0.3 
  1.0.2 
  1.0.1 
  1.0.0
```

##### ISSUE TYPE
- Bug Report

##### STEPS TO REPRODUCE

Follow https://docs.oasis.io/dapp/sapphire/quickstart#init-a-new-hardhat-project

##### ACTUAL RESULTS

Returns error, see above.

##### EXPECTED RESULTS

Should install `@oasisprotocol/sapphire-hardhat` without issues.

**Comment by matevz** [comment]
cc @ptrus

**Comment by CedarMist** [comment]
This should be resolved with sapphire-hardhat v2.19.4 and sapphire-paratime v1.3.0

**Comment by ptrus** [comment]
Worked for me. (re-tried it some hours ago)
